docs/containerd-env-setup.md

```diff
@@ -74,21 +74,23 @@ sudo rm -rf /var/lib/containerd-nydus
 ```
 
 3. Start `containerd-nydus-grpc` (nydus snapshotter):
+Optionally, a TOML nydus-snapshotter configuration file can be provided by appending `--config <CONFIG>` when starting nydus-snapshotter if you want fine-grained control items. An example configuration file can be found [here](https://github.com/containerd/nydus-snapshotter/blob/main/misc/snapshotter/config.toml)
 
 ```bash
 sudo /usr/bin/containerd-nydus-grpc \
-    --config-path /etc/nydus/nydusd-config.fusedev.json \
+    --fs-driver fusedev \
```

Comment:
Can we make it as default?

Reply:
It seems the latest snapshotter must be specified `--address /run/containerd-nydus/containerd-nydus-grpc.sock`.

Reply:
@changweige By the way, when this API changes in nydus-snapshotter, can we give a detailed description when the version is released? Recent API changes are somewhat frequent.

Reply:
> It seems the latest snapshotter must be specified `--address /run/containerd-nydus/containerd-nydus-grpc.sock`.

It has been fixed